src/components/subtitleuploader/subtitleuploader.js

```diff
@@ -0,0 +1,188 @@
+define(['dialogHelper', 'connectionManager', 'dom', 'loading', 'scrollHelper', 'layoutManager', 'globalize', 'require', 'emby-button', 'emby-select', 'formDialogStyle', 'css!./style'], function (dialogHelper, connectionManager, dom, loading, scrollHelper, layoutManager, globalize, require) {
```

Comment:
Any chance you'd be willing to convert this file to ES6 since we're in the middle of a migration right now? No worries if you want to leave that for later.

Reply:
I'm trying. For now I struggle with 'Uncaught ReferenceError: exports is not defined' 😅

Reply:
I think @dmitrylyzo ran into that issue at some point, he might know more.

Reply:
tbh, I don't remember if I had such error. I think that was `default`.

@redSpoutnik Make sure you add your files to babel list in `package.json`.
https://github.com/jellyfin/jellyfin-web/blob/f45a4d1a8ca5d7445a76843d3beae4c5e9947af5/package.json#L91-L113

For more information https://github.com/jellyfin/jellyfin-web/issues/868#issuecomment-593068027

Reply:
Done! It was a mispelling in package.json and a missing export for `function show` :man_facepalming: